src/storage/db_interface_common.py

```diff
@@ -238,6 +239,12 @@ def get_set_of_all_included_files(self, fo):
         else:
             return set()
 
+    def get_uids_of_all_included_files(self, uid: str) -> Set[str]:
```

Comment:
Same size remark from `yara_binary_search.py` would apply here but I don't think there is such a thing as an iterator set ...

Reply:
this should be no issue here, since this is only a set of UIDs. Even a set of 1000 UIDs should take less than 50KB of memory.